tools/system_libs.py

```diff
@@ -1086,7 +1086,7 @@ class libwebgpu_cpp(MTLibrary):
   src_files = ['webgpu_cpp.cpp']
 
 
-class libembind(Library):
```

Comment:
I don't think this should be needed as part of this change.  At least that is not the intention behind the recent llvm changes.

What error were you seeing without this?

Reply:
The usual error on features not matching: can't link in this .o because it disallows atomics.

Reply:
Specifically
```
wasm-ld: error: --shared-memory is disallowed by bind.o because it was not compiled with 'atomics' or 'bulk-memory' features.
```

Reply:
Building `bind.o` with an `-mt` variant fixes it, of course.

What's not clear to me is why this worked before, and what @tlively 's change in LLVM makes different so that it doesn't work now...

Reply:
Hmm I think I know what's going on. I'll do another LLVM patch to explain and fix it.